### DISCUSSION TITLE
Adding  labelList from recharts

### DISCUSSION THREAD

**Discussion by yousefhany77** [mantinedev/mantine#5739] [Feature requests]
# What i want to add 

<img width="1169" alt="Screenshot 2024-02-07 at 5 58 56 PM" src="https://github.com/mantinedev/mantine/assets/59293857/7a42656f-40b8-40a9-90bf-12f527c9d48a">

in the Bar chart component 
passing array of `LabelList` props will render <LabelList/>` components inside the `<Bar/>`

**Comment by nicolasherediaerro** [comment]
You can achive this using css, in my case i'm using tailwind, but you can use css or other css post processor. By using the z-index to modify the placement of the item.

```tsx
<Bar
  dataKey="max"
  stackId="b"
  fill="var(--color-max)"
  unit={chart.units}
  radius={[4, 4, 4, 4]}
>
  <LabelList
    position="centerTop"
    dataKey="max"
    fillOpacity={1}
    className="z-50 fill-black"
  >
    Your custom text
  </LabelList>
  <LabelList position="top" dataKey="max" fillOpacity={1} />
  {chartData.map((item) => (
    <Cell
      key={item.month}
      fill={
        item.max > 0 ? 'var(--color-max)' : 'var(--color-min)'
      }
    />
  ))}
</Bar>
```

![Screenshot 2024-09-05 080717](https://github.com/user-attachments/assets/ff6a562a-28fd-4c56-8492-0b5e4d09a0eb)
